e2e-polybft/e2e/bridge_test.go

```diff
@@ -1250,11 +1250,11 @@ func TestE2E_Bridge_Transfers_AccessLists(t *testing.T) {
 
 func TestE2E_Bridge_NonMintableERC20Token_WithPremine(t *testing.T) {
 	var (
+		epochSize             = 5
```

Comment:
Should we return this back to the 10 maybe (it should give us more chances to catch the issue more often)?

```suggestion
		epochSize             = 10
```